### PR TITLE
EFRS-960 - Update texts on UI.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pbohdan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ui/src/app/features/user-list/user-list.component.html [hunk 1]
```diff
@@ -37,7 +37,6 @@ <h3>{{ 'users.title' | translate }}</h3>
     [availableRoles]="availableRoles$ | async"
     [currentUserId]="currentUserId$ | async"
     [userRole]="userRole$ | async"
-    [createHeader]="messageHeader"
```
Comment (isobolev) [inline ui/src/app/features/user-list/user-list.component.html:40]: will there be modal header when trying to create new user?

Comment (pbohdan) [reply ui/src/app/features/user-list/user-list.component.html:40]: After deleting org_users. add_users_title to en.json this @Input is not used.

Comment (isobolev) [reply ui/src/app/features/user-list/user-list.component.html:40]: can you please delete the   `@Input() createHeader: string;` and its other references in code as well then?

Comment (pbohdan) [reply ui/src/app/features/user-list/user-list.component.html:40]: Done

**Pull request merged**